script/precompile_tasks

```diff
@@ -1,13 +1,9 @@
-#! /bin/bash
-
-# Exit if any subcommand fails
-set -e
-set -o pipefail
+#!/bin/sh -e
 
 # Precompile tasks if not in production and not in CI
 # Also allow skipping precompilation with an ENV var
 if [ "$LUCKY_ENV" != "production" ] && [ -z "$CI" ] && [ -z "$SKIP_LUCKY_TASK_PRECOMPILATION" ]; then
-    shards build && \
-    mkdir -p ../../bin && \
-    cp -r $(pwd)/bin $(pwd)/../..
+    shards build
+    mkdir -p ../../bin
+    cp -r $PWD/bin $PWD/../..
```

Comment:
Again, I thought `pwd` was a standard POSIX thing: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/pwd.html but also, if it works more places, then 👍

Reply:
Yes it is. This change isn't for compatibility, it just avoid to call the `pwd` binary since the `$PWD` variable already exists. That's a minor optimization I've seen.